KarhooUISDK/Screens/CheckoutScreen/NewCheckout/NewCheckoutCoordinator.swift

```diff
@@ -121,6 +124,31 @@ extension KarhooNewCheckoutCoordinator: NewCheckoutRouter {
         viewController.modalPresentationStyle = .overFullScreen
         baseViewController.present(viewController, animated: true, completion: nil)
     }
+    
+    func routeToComment(title: String, comments: String) {
+        guard let presenter = presenter else { return }
```

Comment:
If the guard statement failure is not expected behavior add a `assertionFailure` call, so we are aware of it.